worker/raft/manifold.go

```diff
@@ -131,3 +179,8 @@ type withRaftOutputs interface {
 	Raft() (*raft.Raft, error)
 	LogStore() (raft.LogStore, error)
 }
+
+// NewTarget is a shim to construct a raftlease.NotifyTarget for testability.
```

Comment:
The comment about testability is a little misleading. It's just a late binding for the state reference.

Reply:
Ah, yeah, was a carryover.